tests/integration/Dockerfile.girder_worker

```diff
@@ -0,0 +1,19 @@
+FROM girder/girder_worker:latest
+MAINTAINER Christopher Kotfila <chris.kotfila@kitware.com>
+
+USER root
+
+RUN sed -i  's|^broker.*=.*$|broker = amqp://guest:guest@rabbit/|' /girder_worker/girder_worker/worker.local.cfg
```

Comment:
unfortunately while you can pass ```-b``` on the command line to the girder_worker process,  when you import a task in the girder context to use ```.delay()``` or whatever,  it won't have this  broker,  so it has to be changed in the config file anyways